Feature: Add remove from folder option to apps dashboard

mslourens opened the pull request:
Resolves #1816 

<img width="561" alt="image" src="https://user-images.githubusercontent.com/1907152/155342132-4fb5387b-9aa5-482b-bf70-9e5876fd1348.png">

The menu option is only available when the app is selected inside a folder. When clicked, the app is removed from the folder and the folder is refreshed.

gsmithun4 commented:
Hi @mslourens Can you please add a confirmation modal, you can follow the delete app modal style
![Screenshot 2022-02-24 at 10 34 11 PM](https://user-images.githubusercontent.com/3417097/155572173-7a0eda4d-3100-4667-a7b5-5f7cbfaf422a.png)

mslourens commented:
I moved the handling of the Remove from folder menu item to the more generic `appActionModal` and added a confirmation modal, like so:
<img width="430" alt="image" src="https://user-images.githubusercontent.com/1907152/155608442-5d64e148-cec6-4dc7-96be-9a5eb868796c.png">

gsmithun4 reviewed (approved):
LGTM
App deployed https://tj-remove.herokuapp.com/
